dipper/models/BiolinkVocabulary.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+
+
+class BioLinkVocabulary(Enum):
+    category = "biolink:category"  # this is a node property, not a slot
```

Comment:
This is a node property, not a `class`.
Another name for property is `slot`.

Reply:
okay, thanks, fixed this